src/main/scala/scalismo/faces/momo/MoMoCoefficients.scala

```diff
@@ -17,6 +17,7 @@
 package scalismo.faces.momo
 
 import breeze.linalg.DenseVector
+import breeze.stats.distributions.Gaussian
```

Comment:
If you do not move the functionality from MoMoInstance to here then this import is not needed.